style.css

```diff
@@ -0,0 +1,146 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@300;400;500;700;900&display=swap");
+
+body {
+  margin: 0;
+  width: 338px;
+  height: 675px;
+  border: 1px solid lightgrey;
+  overscroll-behavior-y: scroll;
+}
+
+/* nav {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  height: 46px;
+  width: 375px;
+}
+
+.time {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 54px;
+  height: 16px;
+  border: 1px solid lightgrey;
+  margin-left: 21px;
+}
+
+.timeparagraph {
+  font-size: 14px;
+  font-weight: 500;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+}
+
+.icons {
+  margin-right: 16px;
+  border: 1px solid black;
+  height: 16px;
+} */
+
+/* .container {
+  height: 672px;
+  width: 375px;
+  border: 1px solid red;
+} */
+
+/* =====LOGO-MENU==== */
+div.logo-menu {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 338px;
+  height: 46px;
+  top: 0;
+}
+
+p.logo {
+  margin-left: 12px;
+  font-family: "Poppins", Arial, sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.fa-bars {
+  margin-right: 24px;
+  color: #6070ff;
+  height: 24px;
+  width: 24px;
+  margin-top: 8px;
+}
+
+/* =====MAIN BODY======= */
+
+.main-container {
+  position: fixed;
+  margin-top: 46px;
+  width: 338px;
+  height: 675px;
+  background-image: url("./images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 338px 538px;
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  width: 300px;
+  height: 150px;
+  font-family: "Poppins", Arial, sans-serif;
+  font-weight: 700;
+  font-size: 25px;
+  color: #172b4d;
+  margin-left: 24px;
+  margin-top: 114px;
+  line-height: 52px;
+}
+
+.titleText {
+  display: flex;
+  align-items: center;
+  width: 300px;
+  height: 113px;
+  font-family: "Poppins", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  color: #172b4d;
+  margin-left: 24px;
+  margin-top: 6px;
+  line-height: 20px;
+}
+
+/* =====FOOTER==== */
+
+.connect p {
+  display: flex;
+  width: 119px;
+  height: 24px;
+  font-family: "Poppins", Arial, sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-left: 24px;
+  margin-top: 6px;
+}
+
+.footer-icons {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  width: 190px;
+  height: 30px;
+  margin-left: 24px;
+  margin-top: 6px;
+}
+
+.footer-icons img {
+  width: 25px;
+  height: 25px;
+  color: #505f79;
+}
```

Comment:
- [ ] As stated the Project requirement Kindly make sure that You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).So Kindly Adjust the right alignment of the your Toolbar and Headline section Just Like the Figma Design for Minimum mobile screen size should be 375px.

- The Figma design For Minimum mobile screen size should be 375px

![Capture](https://user-images.githubusercontent.com/93598189/212900374-6c1ced29-0171-4b1e-92b7-27758ca03836.PNG)

- Your Project Look likes this : Minimum mobile screen size should be 375px

![Capture](https://user-images.githubusercontent.com/93598189/212900708-07907716-c790-49cd-bf0d-6171d54c5b2a.PNG)